src/BloomBrowserUI/bookEdit/OverflowChecker/OverflowChecker.ts

```diff
@@ -13,6 +13,9 @@ class OverflowChecker {
         var queryElementsThatCanOverflow = ".bloom-editable, textarea";
         var editablePageElements = $(container).find(queryElementsThatCanOverflow);
 
+        // BL-1260: disable overflow checking for pages with too many elements
+        if (editablePageElements.length > 30) return;
+
```

Comment:
Seems rather arbitrary...and a page with lots of little blocks is just where overflow checking would be most helpful.

Could we suppress it more selectively? For example, maybe we could at least check and correct overflow indication for the block actually being edited and the page as a whole?